src/client/utils/tracking/trackSignedCustomerEvent.ts

```diff
@@ -0,0 +1,46 @@
+import { QuoteBundle } from 'data/graphql'
+import { Variation } from 'utils/hooks/useVariation'
+import { quoteBundleMainQuoteSelector } from 'api/quoteBundleMainQuoteSelector'
+import { handleSignedEvent } from './signing'
+import { trackOfferEvent } from './trackOfferEvent'
+import { EventName } from './gtm'
+import { adtractionQuoteCart } from './adtractionQuoteCart'
+
+export type TrackSignedEventParams = {
+  variation: Variation | null
+  quoteCartId: string
+  memberId: string
+  bundle: QuoteBundle
+  campaignCode?: string
+  isDiscountMonthlyCostDeduction: boolean
+}
+
+export const trackSignedCustomerEvent = ({
+  variation,
+  quoteCartId,
+  memberId,
+  bundle,
+  campaignCode,
+  isDiscountMonthlyCostDeduction,
+}: TrackSignedEventParams) => {
+  const mainQuote = quoteBundleMainQuoteSelector(bundle)
+
+  if (variation === Variation.AVY) {
+    handleSignedEvent(memberId)
+  }
```

Comment:
Should we check whether we really need to keep all of these special Avy things in our codebase? From what I understand they _should_ only use our API's directly now and not the web onboarding.